src/cluecode/finder.py

```diff
@@ -60,6 +75,9 @@ def find(location, patterns):
 
     for i, line in enumerate(analysis.text_lines(location)):
         lineno = i + 1
+        if (patterns[0][0] in ['emails', 'urls']) and (
```

Comment:
`patterns[0][0]` sounds like a really dangerous statement to me and is rather obscure: what does it means?

Reply:
Fixed, thanks ! :)